### PR TITLE
Cow Game modal component re-style.

### PR DESCRIPTION
**Pull request by biocom**
- Restyles the tx. confirmation modal to show the CowGame button:

Screen Shot 2021-07-05 at 16 38 12
Screen Shot 2021-07-05 at 16 37 33
Screen Shot 2021-07-05 at 16 37 20
Screen Shot 2021-07-05 at 16 37 08

### CODE REVIEW

File: src/custom/components/TransactionConfirmationModal/index.tsx [hunk 1]
```diff
@@ -1,4 +1,175 @@
...
+import MetaMaskLogo from 'assets/images/metamask.png'
+import { getEtherscanLink, getExplorerLabel } from 'utils'
+import { Text } from 'rebass'
+import { ArrowUpCircle, CheckCircle } from 'react-feather'
+import useAddTokenToMetamask from 'hooks/useAddTokenToMetamask'
+import GameIcon from 'assets/cow-swap/game.gif'
+import { Link } from 'react-router-dom'
+
 // Modified only for adding a better label for the Etherscan link
 //  getExplorerLabel(chainId, hash, 'transaction')
```
Comment (alfetopito) [inline src/custom/components/TransactionConfirmationModal/index.tsx:17]: No longer true

```suggestion
```

**Review by alfetopito — COMMENTED** [review]
Found a weird bug.

screenshot_2021-07-05_13-16-40

## Steps:
1. Open the app at any page https://pr856--gpswapui.review.gnosisdev.com/
2. Open the game using the menu
3. Go back to swap page (click on header)
4. Open the game using the menu again

Cow is huge, game doesn't start.

Also, the text jumping still happen every time I start the game (when "Press space to start game" message disappears)

**Comment by biocom** [comment]
> Found a weird bug.
> 
> screenshot_2021-07-05_13-16-40
> 
> ## Steps:
> 1. Open the app at any page https://pr856--gpswapui.review.gnosisdev.com/
> 2. Open the game using the menu
> 3. Go back to swap page (click on header)
> 4. Open the game using the menu again
> 
> Cow is huge, game doesn't start.
> 
> Also, the text jumping still happen every time I start the game (when "Press space to start game" message disappears)

Good find. Created an issue for this https://github.com/gnosis/cowswap/issues/858

**Review by anxolin — APPROVED** [review]
Looks great. 

ubernit: I see that "add token to MM" functionality closes the modal. I think is better if it doesn't do that, right? If you want to add to MM and play the game, you can't. 
not super important though.

**Review by alfetopito — APPROVED** [review]
Since the issue is unrelated, approving

**Review by W3stside — APPROVED** [review]
is the base branch correct?

**Comment by biocom** [comment]
> Looks great.
> 
> ubernit: I see that "add token to MM" functionality closes the modal. I think is better if it doesn't do that, right? If you want to add to MM and play the game, you can't.
> not super important though.

Doesn't happen for me on Brave. What browser you used? I see this after adding the token to MM:
Screen Shot 2021-07-07 at 13 44 49

**Pull request merged**